sdk/core/azure-core/inc/http/curl/curl.hpp

```diff
@@ -12,31 +12,100 @@
 #include <type_traits>
 #include <vector>
 
+#define LIBCURL_READER_SIZE 100
+#define UPLOAD_STREAM_PAGE_SIZE 1024 * 64
```

Comment:
Macros are not azure-prefixed, but we chould not be using them anyway. Use `constexpr`.

Reply:
request